include/eve/module/core/regular/scatter.hpp

```diff
@@ -62,6 +64,23 @@ struct scatter_t : callable<scatter_t, Options, relative_conditional_no_alternat
 //!     * `ptr` : Base pointer to scatter to.
 //!     * `idx` :  [Integral SIMD value](@ref eve::integral_simd_value) containing the index to scatter to.
 //!
+//!   @groupheader{Semantic Modifiers}
+//!
+//!   * Masked Call
+//!
+//!   eve::scatter can be masked using [Relative conditionals](@ref relative_conditional_expr) to skip scattering
+//!   of certain elements.
+//!
+//!   The call `eve::scatter[cond](v,p,i)` is semantically equivalent to:
+//!
+//!   @code
+//!   auto m = cond.mask( as<as_logical_t<T>>{} );
+//!   for(std::size_t n=0;n<v.size();++n)
+//!   {
+//!     if(m.get(n)) write(v.get(n),p+idx.get(n));
+//!   }
+//!   @endcode
```

Comment:
We need to decide how we do this.

Because I was doing it like:

```
auto compress_store
       [safe/unsafe]
       [C1 ignore_in][C2 ignore_out]
```

https://jfalcou.github.io/eve/group__core__compress_gaa538588c547767772b1b1fe4b61f357e.html#gaa538588c547767772b1b1fe4b61f357e

Reply:
Let just decide that later. I Still need to do the SVE scatter so we have one more shot ;)